modules/lmdbbackend/lmdbbackend.cc

```diff
@@ -982,7 +982,7 @@ bool LMDBBackend::createDomain(const DNSName& domain, const DomainInfo::DomainKi
   return true;
 }
 
-void LMDBBackend::getAllDomains(vector<DomainInfo>* domains, bool include_disabled)
+void LMDBBackend::getAllDomains(vector<DomainInfo>* domains, bool doSerial, bool include_disabled)
```

Comment:
```suggestion
void LMDBBackend::getAllDomains(vector<DomainInfo>* domains, bool getSerial, bool include_disabled)
```

Reply:
There is a getSerial function in LMDBBackend